streams/src/main/java/org/apache/kafka/streams/state/internals/StoreChangeLogger.java

```diff
@@ -55,7 +55,7 @@ void logChange(final K key, final V value) {
         if (collector != null) {
             final Serializer<K> keySerializer = serialization.keySerializer();
             final Serializer<V> valueSerializer = serialization.valueSerializer();
-            collector.send(this.topic, key, value, this.partition, context.timestamp(), keySerializer, valueSerializer);
+            collector.send(this.topic, key, value, context.headers(), this.partition, context.timestamp(), keySerializer, valueSerializer);
```

Comment:
Aren't we going to not send the headers to changelog topics??

Reply:
You're right. Fixing it